tests/ui/invalid_pymethod_names.stderr

```diff
@@ -1,8 +1,8 @@
 error: `name` may only be specified once
-  --> tests/ui/invalid_pymethod_names.rs:10:19
+  --> tests/ui/invalid_pymethod_names.rs:11:14
    |
-10 |     #[pyo3(name = "num")]
-   |                   ^^^^^
+11 |     #[getter(number)]
+   |              ^^^^^^
```

Comment:
This test changed from erroring on the first of two attributes which specify name (`#[pyo3(name = "num")]` to erroring on the second site `(`#[getter(number)]`). Functionally no different as the code is rejected either way.